src/components/tasks/task.js

```diff
@@ -688,6 +689,10 @@ const createTaskInput = (name = '', difficulty = '', isEdit, task = null) => {
         alert('Please fill all the fields.');
         return;
       }
+      if (timeHourInput < 0 || timeMinInput < 1) {
```

Comment:
minutes can be 0 as long as hours isn't also 0

Reply:
I changed it in main earlier but I think someone deleted it. Change it to the following:
if ((timeHourInput < 1 && timeMinInput < 1) || (timeHourInput > 0 && timeMinInput < 0))

Reply:
Fixed it.